src/rulesets/rest/2022-05-25/utils.ts

```diff
@@ -61,3 +61,7 @@ export const isResourceMetaProperty = (property: Field): boolean => {
     ) !== null;
   return isResourceMetaProperty;
 };
+
+export const specIsRemoved = (spec): boolean => {
+  return spec.change === "removed";
```

Comment:
Does this apply to the entire specification being deleted? Or the entity being evaluated?

Reply:
The spec being passed in is the spec itself. The whole thing has to be deleted for it to be ok for breaking change deletes to happen, since the lifecycle management of the spec is handled in another rule.